server/asset/eth/common.go

```diff
@@ -226,6 +227,16 @@ func decodeAmountCoinID(coinID []byte) (*AmountCoinID, error) {
 	}, nil
 }
 
+func CreateAmountCoinID(address common.Address, amount uint64) *AmountCoinID {
+	var nonce [8]byte
+	copy(nonce[:], encode.RandomBytes(8))
+	return &AmountCoinID{
+		Address: address,
+		Amount:  amount,
+		Nonce:   nonce,
+	}
+}
+
```

Comment:
As per convo elsewhere, we will probably only use this client-side. Can move and unexport here or separately. Up to you.

Reply:
I'll do it separately.